integrations/symfony-bundle.rst

```diff
@@ -357,6 +357,32 @@ You can configure a client with authentication. Valid authentication types are
     Using query parameters for authentication is :ref:`not safe <Authentication-QueryParams>`.
     The auth params will appear on the URL and we recommend to NOT log your request, especially on production side.
 
+VCR Plugin
+``````````
+
+Allow you to enable the :doc:`VCR Plugin </plugins/vcr>` and configure the behavior to use ( Replay, Record , Replay OR Record).
+
+.. code-block:: yaml
+
+    // config.yml
+    httplug:
+        clients:
+            acme:
+                plugins:
+                - vcr:
+                    mode: replay # record | replay | replay_or_record
+                    recorder: filesystem # optional
```

Comment:
i'd put this commented out to make it clear its optional.

Reply:
Fixed, as well as the above